annotator/src/main/java/org/cbioportal/models/MutationRecord.java

```diff
@@ -507,4 +509,61 @@ public void initHeader() {
         header.add("n_ref_count");
         header.add("n_alt_count");
     }
+
+    private static List<String> initMissingValueList() {
+        List<String> missingValueList = new ArrayList<>();
+        missingValueList.add("NA");
+        missingValueList.add("N/A");
+        missingValueList.add("N/a");
+        missingValueList.add("n/A");
```

Comment:
I think it would be better to have an enum or something, or to do this check in a case insensitive way.